ios/gamma/split-view/RNSSplitViewScreenController.swift

```diff
@@ -18,8 +18,14 @@ public class RNSSplitViewScreenController: UIViewController {
     return splitViewScreenComponentView.reactEventEmitter()
   }
 
+  private var displayLink: CADisplayLink?
+  private var lastAnimationFrame: CGRect?
+  private var transitionInProgress: Bool
```

Comment:
It would be nice to declare some inline struct / class and group this state together. I'm unsure whether this is possible in Swift. If not, just create a file-private type to group this.

Reply:
e.g. `SizeTransitionTrackingState` or something.